sgftools/annotations.py

```diff
@@ -115,32 +113,56 @@ def format_analysis(stats, move_list, this_move):
 
         for move_label, move in list(zip(abet, move_list)):
             comment += "%s -> Win%%: %.2f%% (%d visits) \n" \
-                       % (move_label, flip_winrate(move['winrate'], move['color']) * 100, move['visits'])
+                       % (move_label, move['winrate'] * 100, move['visits'])
+
+    # Mark labels and skip passes
+    label_values = []
+    for move_label, move in zip(abet, move_list):
+        if move['pos'] not in ["", "tt"]:
+            label_values.append("%s:%s" % (move['pos'], move_label))
+
+    suggested_moves = [move['pos'] for move in move_list]
+
+    # Mark triangles
+    if this_move and this_move not in suggested_moves and not pos_is_pass(this_move):
+        triangle_values = [this_move]
+    else:
+        triangle_values = None
+
+    return comment, label_values, triangle_values
 
-    # Check for pos being "" or "tt", values which indicate passes, and don't attempt to display markers for them
-    LB_values = ["%s:%s" % (mv['pos'], L) for L, mv in zip(abet, move_list) if mv['pos'] != "" and mv['pos'] != "tt"]
-    mvs = [mv['pos'] for mv in move_list]
-    TR_values = [this_move] if this_move not in mvs and this_move is not None and not pos_is_pass(this_move) else []
-    return comment, LB_values, TR_values
 
+def annotate_sgf(cursor, comment, labels_values=None, triangle_values=None):
+    """
+    Add comment, labels and triangles to node
+    """
+    node_comment = cursor.node.get('C')
```

Comment:
how about move all marks related to sgf format, into separate sgf_specification file with explanation of meaning of each?
I found it annoying to check each time sgf official documentation. Its not very clear orginized/explained.